### PR TITLE
Update Developer Guide

### PR DESCRIPTION
**Pull request by jjlee050**
- Update user authentication implementation.
- Add about this guide
- Edit prerequisites

Signed-off-by: jjlee050 <josephlee050@gmail.com>

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -16,10 +16,19 @@ endif::[]
 
 By: `W14-1`      Since: `Sept 2018`      Licence: `MIT`
 
+= About this guide
+
+This guide provides an overview of ClinicIO and walks you through the installation process.
+In addition, this guide provides in-depth documentation for all features.
+
+This guide assume that readers has certain developers knowledge.
```
Comment (arsalanc-v2) [inline docs/DeveloperGuide.adoc]: I think you should qualify this or remove it

Comment (arsalanc-v2) [reply docs/DeveloperGuide.adoc]: Briefly explain what you mean by a developer having a certain amount of knowledge. 

How much should they know? 
What should they know?

Comment (jjlee050) [reply docs/DeveloperGuide.adoc]: Oh k. Will work on it.

**Comment by jjlee050** [comment]
Hang on. I work on the about this guide.

**Pull request merged**